src/App.js

```diff
@@ -1,30 +1,40 @@
-import React, { useState, useEffect } from 'react';
-import { Routes, Route } from 'react-router-dom';
-import { getAllData } from './util/index';
-import { HomePage } from './components/HomePage/HomePage';
-import { LoginPage } from './components/LoginPage/LoginPage';
-import ProtectedRoute from './components/ProtectedRoute/ProtectedRoute';
```

Comment:
This all needs to still be imported (other than the duplicate react import. I think this got deleted during the merge conflict. In this case you probably wanted to accept both changes and then delete the double import.